### PR TITLE
Add file encryption using box and the enclave's public key

### LINKED ISSUE #2: Create page for uploading data files
- [x] Encrypt the file selected for upload (use any generated key for now)

### LINKED ISSUE #8: Use the public key from the `x-ms-sgx-ehd` field of the attestation JWT to encrypt the data for the enclave
As part of the encryption the application needs to:
  - [x] Generate an ephemeral public key
  - [ ] Pass any metadata through as associated data (Not sure if tweetnacl supports associated data, so holding off on this for now)
  - [x] after encryption, the metadata needs to be presented along with the file for uploading

### LINKED ISSUE #9: Send the encrypted payload to the backend, and display the response values
Note: Rename `password` to `access-key`.

The response will be encrypted with the provided ephemeral key, and needs to be decrypted on the client side. The ephemeral key should be discarded after decrypting the payload.

The response should contain the unique key referencing the uploaded data, as well as the "password" that can be used to issue execution tokens in the future.

Both of these values need to be made available to the user, either via a file download or displayed on the UI.

### PR DESCRIPTION
**Pull request by longtomjr**
closes: #8

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
             <div class="card-header">
               <span>JWT Token</span>
               <span>
-                <el-button size="small" @click="fetchJWT()"> Fetch </el-button>
+                <el-button size="small" @click="getAttestation()"> Fetch </el-button>
```
Comment (jongbonga) [inline src/views/Home.vue]: @longtomjr should we remove the "validate" button that was previously getting the attestation?

**Comment by PiDelport** [comment]
@jongbonga: It looks like the merge accidentally undid commit c3f0317… I'll push a rebase that includes that, and we can go over it?

**Review by PiDelport — COMMENTED** [review]
Looks good!

I added commits for the issues I spotted: can you check that it looks good?

**Review by PiDelport — APPROVED** [review]

File: src/components/FileUpload.vue [hunk 2]
```diff
@@ -37,6 +39,12 @@
           <el-descriptions-item label="Extension">
             {{ state._file.extension }}
           </el-descriptions-item>
+          <el-descriptions-item label="Upload password">
```
Comment (longtomjr) [inline src/components/FileUpload.vue]: Rename "password" to "access-key"

**Review by jongbonga — APPROVED** [review]

**Pull request merged**